activerecord/test/cases/persistence_test.rb

```diff
@@ -1077,6 +1077,15 @@ def test_update_column
     assert_not_predicate topic, :approved?
   end
 
+  def test_update_column_touch_option
+    topic = Topic.find(1)
+    updated_at = topic.updated_at
+    travel(1.second) do
+      topic.update_column(:title, "super_title", touch: true)
+    end
+    assert_not_equal updated_at, topic.updated_at
```

Comment:
I think, in this and the following tests we can utilise the `assert_changes` test helper.